packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -486,7 +497,12 @@ const stepFlow = async (
       case 'withdrawToSeat': {
         const amounts = { Cash: amount };
         todo.push(async () => {
-          const { lca } = await provideCosmosAccount(orch, 'agoric', kit);
+          const { lca } = await provideCosmosAccount(
+            orch,
+            'agoric',
+            kit,
+            portfolioTrace,
```

Comment:
```suggestion
            indexTrace,
```

(or moveTrace)

and likewise for the rest of the scope of the for loop